frontend/src/pages/settings/logs-and-activity.vue

```diff
@@ -1,112 +1,103 @@
 <template>
-  <settings-page page-title="settingsSections.logsAndActivity.logsAndActivity">
+  <settings-page page-title="settingsSections.logs.name">
     <template #content>
       <v-col md="6" class="pt-0 pb-4">
         <v-fade-transition group>
           <h2 key="logs-title" class="text-h6 mb-2">
-            {{ $t('settings.logsAndActivity.logs') }}
+            {{ t('settings.logsAndActivity.logs') }}
           </h2>
           <v-list
-            v-if="logFiles && logFiles.length > 0"
+            v-if="logs.status === 'loaded' && logs.data.length > 0"
             key="log-list"
             lines="two"
             class="mb-2">
-            <v-list-group>
-              <v-list-item
-                v-for="file in logFiles"
-                :key="file.Name"
-                :href="getLogFileLink(file.Name)"
-                target="_blank"
-                rel="noopener">
+            <v-list-item
+              v-for="file in logs.data"
+              :key="file.Name ?? undefined"
+              :href="getLogFileLink(file.Name ?? '')"
```

Comment:
The html helpers only have html sanitization, this a url. SonarCloud doesn't seem to have any issues, and I think that this call is fine especially since the user supplied data is templated as a query param so there's no sanitization needed in this case.